docs/ref/urlresolvers.txt

```diff
@@ -171,6 +171,20 @@ If the URL does not resolve, the function raises a
         The name of the view that matches the URL, including the namespace if
         there is one.
 
+    .. attribute:: ResolverMatch.captured_kwargs
+
+        .. versionadded:: 4.1
+
+        The captured keyword arguments that would be passed to the view
+        function via route argument, as parsed from the URL.
+
+    .. attribute:: ResolverMatch.extra_kwargs
+
+        .. versionadded:: 4.1
+
+        The additional keyword arguments that would be passed to the view
+        function via kwargs argument, as parsed from the URL.
```

Comment:
:thinking: 
```suggestion
        function.
```

Reply:
Should we update an example?:
https://github.com/django/django/blob/abfdb4d7f384fb06ed9b7ca37b548542df7b5dda/docs/ref/urlresolvers.txt#L186-L206